internal/handshake/transport_parameters.go

```diff
@@ -204,6 +205,15 @@ func (p *TransportParameters) Marshal() []byte {
 	b := &bytes.Buffer{}
 	b.Write([]byte{0, 0}) // length. Will be replaced later
 
+	// add a greased value
+	utils.BigEndian.WriteUint16(b, uint16(27+31*rand.Intn(100)))
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Seed once, e.g. in init(). Seeding this way may be expensive depending on the implementation.